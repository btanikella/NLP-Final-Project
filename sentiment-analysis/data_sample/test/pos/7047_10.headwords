saw
gotten
like
was
enjoy
full
recommend
