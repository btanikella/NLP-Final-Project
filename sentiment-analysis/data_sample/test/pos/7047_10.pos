PRP RB VBD DT NN IN DT JJ CD POS CC VBD PRP VBD DT JJ NN
IN PRP VBP VBN JJR PRP VBP DT NN IN JJ NNS
PRP VBP TO VB PRP WRB PRP VBP VBG JJ IN PRP$  VBN NN
PRP VBD DT NN -LRB- CD -RRB- IN DT NN IN DT NN IN DT NN IN PRP MD VB IN JJ IN DT JJ NNS IN DT NN POS IN DT NN VBP
PRP VBP VBG DT NNS WDT VBD VBN IN DT NN CC VBG DT JJ JJ JJ NNS JJ IN VBD VBN IN DT JJ NN NN
DT NN VBZ RB JJ IN JJ CD NNS
PRP RB VB DT NN TO NN WDT VBD DT NN IN DT JJ CD POS TO JJ CD POS
