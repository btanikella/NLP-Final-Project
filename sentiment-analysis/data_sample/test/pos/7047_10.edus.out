Sentence #1 (18 tokens):
I first saw this movie in the mid 80's and thought it was a funny movie.
[Text=I CharacterOffsetBegin=4 CharacterOffsetEnd=5 PartOfSpeech=PRP] [Text=first CharacterOffsetBegin=6 CharacterOffsetEnd=11 PartOfSpeech=RB] [Text=saw CharacterOffsetBegin=12 CharacterOffsetEnd=15 PartOfSpeech=VBD] [Text=this CharacterOffsetBegin=16 CharacterOffsetEnd=20 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=21 CharacterOffsetEnd=26 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=27 CharacterOffsetEnd=29 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=30 CharacterOffsetEnd=33 PartOfSpeech=DT] [Text=mid CharacterOffsetBegin=34 CharacterOffsetEnd=37 PartOfSpeech=JJ] [Text=80 CharacterOffsetBegin=38 CharacterOffsetEnd=40 PartOfSpeech=CD] [Text='s CharacterOffsetBegin=40 CharacterOffsetEnd=42 PartOfSpeech=POS] [Text=and CharacterOffsetBegin=43 CharacterOffsetEnd=46 PartOfSpeech=CC] [Text=thought CharacterOffsetBegin=47 CharacterOffsetEnd=54 PartOfSpeech=VBD] [Text=it CharacterOffsetBegin=55 CharacterOffsetEnd=57 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=58 CharacterOffsetEnd=61 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=62 CharacterOffsetEnd=63 PartOfSpeech=DT] [Text=funny CharacterOffsetBegin=64 CharacterOffsetEnd=69 PartOfSpeech=JJ] [Text=movie CharacterOffsetBegin=70 CharacterOffsetEnd=75 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=75 CharacterOffsetEnd=76 PartOfSpeech=.] 
root(ROOT-0, saw-3)
nsubj(saw-3, I-1)
advmod(saw-3, first-2)
det(movie-5, this-4)
dobj(saw-3, movie-5)
prep(saw-3, in-6)
det(80-9, the-7)
amod(80-9, mid-8)
pobj(in-6, 80-9)
possessive(80-9, 's-10)
cc(saw-3, and-11)
conj(saw-3, thought-12)
nsubj(movie-17, it-13)
cop(movie-17, was-14)
det(movie-17, a-15)
amod(movie-17, funny-16)
ccomp(thought-12, movie-17)
punct(saw-3, .-18)

Sentence #2 (13 tokens):
As I have gotten older I watch this movie for different reasons.
[Text=As CharacterOffsetBegin=81 CharacterOffsetEnd=83 PartOfSpeech=IN] [Text=I CharacterOffsetBegin=84 CharacterOffsetEnd=85 PartOfSpeech=PRP] [Text=have CharacterOffsetBegin=86 CharacterOffsetEnd=90 PartOfSpeech=VBP] [Text=gotten CharacterOffsetBegin=91 CharacterOffsetEnd=97 PartOfSpeech=VBN] [Text=older CharacterOffsetBegin=98 CharacterOffsetEnd=103 PartOfSpeech=JJR] [Text=I CharacterOffsetBegin=104 CharacterOffsetEnd=105 PartOfSpeech=PRP] [Text=watch CharacterOffsetBegin=106 CharacterOffsetEnd=111 PartOfSpeech=VBP] [Text=this CharacterOffsetBegin=112 CharacterOffsetEnd=116 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=117 CharacterOffsetEnd=122 PartOfSpeech=NN] [Text=for CharacterOffsetBegin=123 CharacterOffsetEnd=126 PartOfSpeech=IN] [Text=different CharacterOffsetBegin=127 CharacterOffsetEnd=136 PartOfSpeech=JJ] [Text=reasons CharacterOffsetBegin=137 CharacterOffsetEnd=144 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=144 CharacterOffsetEnd=145 PartOfSpeech=.] 
root(ROOT-0, gotten-4)
mark(gotten-4, As-1)
nsubj(gotten-4, I-2)
aux(gotten-4, have-3)
advmod(watch-7, older-5)
nsubj(watch-7, I-6)
ccomp(gotten-4, watch-7)
det(movie-9, this-8)
dobj(watch-7, movie-9)
prep(watch-7, for-10)
amod(reasons-12, different-11)
pobj(for-10, reasons-12)
punct(gotten-4, .-13)

Sentence #3 (17 tokens):
I like to view it when I'm feeling sad/nostalgic for my "lost youth".
[Text=I CharacterOffsetBegin=150 CharacterOffsetEnd=151 PartOfSpeech=PRP] [Text=like CharacterOffsetBegin=152 CharacterOffsetEnd=156 PartOfSpeech=VBP] [Text=to CharacterOffsetBegin=157 CharacterOffsetEnd=159 PartOfSpeech=TO] [Text=view CharacterOffsetBegin=160 CharacterOffsetEnd=164 PartOfSpeech=VB] [Text=it CharacterOffsetBegin=165 CharacterOffsetEnd=167 PartOfSpeech=PRP] [Text=when CharacterOffsetBegin=168 CharacterOffsetEnd=172 PartOfSpeech=WRB] [Text=I CharacterOffsetBegin=173 CharacterOffsetEnd=174 PartOfSpeech=PRP] [Text='m CharacterOffsetBegin=174 CharacterOffsetEnd=176 PartOfSpeech=VBP] [Text=feeling CharacterOffsetBegin=177 CharacterOffsetEnd=184 PartOfSpeech=VBG] [Text=sad/nostalgic CharacterOffsetBegin=185 CharacterOffsetEnd=198 PartOfSpeech=JJ] [Text=for CharacterOffsetBegin=199 CharacterOffsetEnd=202 PartOfSpeech=IN] [Text=my CharacterOffsetBegin=203 CharacterOffsetEnd=205 PartOfSpeech=PRP$] [Text=`` CharacterOffsetBegin=206 CharacterOffsetEnd=207 PartOfSpeech=``] [Text=lost CharacterOffsetBegin=207 CharacterOffsetEnd=211 PartOfSpeech=VBN] [Text=youth CharacterOffsetBegin=212 CharacterOffsetEnd=217 PartOfSpeech=NN] [Text='' CharacterOffsetBegin=217 CharacterOffsetEnd=218 PartOfSpeech=''] [Text=. CharacterOffsetBegin=218 CharacterOffsetEnd=219 PartOfSpeech=.] 
root(ROOT-0, like-2)
nsubj(like-2, I-1)
aux(view-4, to-3)
xcomp(like-2, view-4)
dobj(view-4, it-5)
advmod(feeling-9, when-6)
nsubj(feeling-9, I-7)
aux(feeling-9, 'm-8)
advcl(view-4, feeling-9)
acomp(feeling-9, sad/nostalgic-10)
prep(sad/nostalgic-10, for-11)
pobj(for-11, my-12)
punct(lost-14, ``-13)
dep(my-12, lost-14)
dobj(lost-14, youth-15)
punct(lost-14, ''-16)
punct(like-2, .-17)

Sentence #4 (35 tokens):
I was a teenager (14) at the time of the release of this film so I can identify with many of the same problems/dilemmas that the campers/CIT's in this movie have.
[Text=I CharacterOffsetBegin=224 CharacterOffsetEnd=225 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=226 CharacterOffsetEnd=229 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=230 CharacterOffsetEnd=231 PartOfSpeech=DT] [Text=teenager CharacterOffsetBegin=232 CharacterOffsetEnd=240 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=241 CharacterOffsetEnd=242 PartOfSpeech=-LRB-] [Text=14 CharacterOffsetBegin=242 CharacterOffsetEnd=244 PartOfSpeech=CD] [Text=-RRB- CharacterOffsetBegin=244 CharacterOffsetEnd=245 PartOfSpeech=-RRB-] [Text=at CharacterOffsetBegin=246 CharacterOffsetEnd=248 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=249 CharacterOffsetEnd=252 PartOfSpeech=DT] [Text=time CharacterOffsetBegin=253 CharacterOffsetEnd=257 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=258 CharacterOffsetEnd=260 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=261 CharacterOffsetEnd=264 PartOfSpeech=DT] [Text=release CharacterOffsetBegin=265 CharacterOffsetEnd=272 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=273 CharacterOffsetEnd=275 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=276 CharacterOffsetEnd=280 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=281 CharacterOffsetEnd=285 PartOfSpeech=NN] [Text=so CharacterOffsetBegin=286 CharacterOffsetEnd=288 PartOfSpeech=IN] [Text=I CharacterOffsetBegin=289 CharacterOffsetEnd=290 PartOfSpeech=PRP] [Text=can CharacterOffsetBegin=291 CharacterOffsetEnd=294 PartOfSpeech=MD] [Text=identify CharacterOffsetBegin=295 CharacterOffsetEnd=303 PartOfSpeech=VB] [Text=with CharacterOffsetBegin=304 CharacterOffsetEnd=308 PartOfSpeech=IN] [Text=many CharacterOffsetBegin=309 CharacterOffsetEnd=313 PartOfSpeech=JJ] [Text=of CharacterOffsetBegin=314 CharacterOffsetEnd=316 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=317 CharacterOffsetEnd=320 PartOfSpeech=DT] [Text=same CharacterOffsetBegin=321 CharacterOffsetEnd=325 PartOfSpeech=JJ] [Text=problems/dilemmas CharacterOffsetBegin=326 CharacterOffsetEnd=343 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=344 CharacterOffsetEnd=348 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=349 CharacterOffsetEnd=352 PartOfSpeech=DT] [Text=campers/CIT CharacterOffsetBegin=353 CharacterOffsetEnd=364 PartOfSpeech=NN] [Text='s CharacterOffsetBegin=364 CharacterOffsetEnd=366 PartOfSpeech=POS] [Text=in CharacterOffsetBegin=367 CharacterOffsetEnd=369 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=370 CharacterOffsetEnd=374 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=375 CharacterOffsetEnd=380 PartOfSpeech=NN] [Text=have CharacterOffsetBegin=381 CharacterOffsetEnd=385 PartOfSpeech=VBP] [Text=. CharacterOffsetBegin=385 CharacterOffsetEnd=386 PartOfSpeech=.] 
root(ROOT-0, was-2)
nsubj(was-2, I-1)
dep(was-2, a-3)
amod(a-3, teenager-4)
punct(14-6, -LRB--5)
dep(teenager-4, 14-6)
punct(14-6, -RRB--7)
prep(teenager-4, at-8)
det(time-10, the-9)
pobj(at-8, time-10)
prep(time-10, of-11)
det(release-13, the-12)
pobj(of-11, release-13)
prep(release-13, of-14)
det(film-16, this-15)
pobj(of-14, film-16)
mark(identify-20, so-17)
nsubj(identify-20, I-18)
aux(identify-20, can-19)
advcl(teenager-4, identify-20)
prep(identify-20, with-21)
pobj(with-21, many-22)
prep(many-22, of-23)
det(problems/dilemmas-26, the-24)
amod(problems/dilemmas-26, same-25)
pobj(of-23, problems/dilemmas-26)
dobj(have-34, that-27)
det(campers/CIT-29, the-28)
nsubj(have-34, campers/CIT-29)
possessive(campers/CIT-29, 's-30)
prep(campers/CIT-29, in-31)
det(movie-33, this-32)
pobj(in-31, movie-33)
rcmod(problems/dilemmas-26, have-34)
punct(was-2, .-35)

Sentence #5 (28 tokens):
I enjoy seeing the clothes that were worn at this time and hearing the same old urban legends such as was told on the overnight canoe trip.
[Text=I CharacterOffsetBegin=391 CharacterOffsetEnd=392 PartOfSpeech=PRP] [Text=enjoy CharacterOffsetBegin=393 CharacterOffsetEnd=398 PartOfSpeech=VBP] [Text=seeing CharacterOffsetBegin=399 CharacterOffsetEnd=405 PartOfSpeech=VBG] [Text=the CharacterOffsetBegin=406 CharacterOffsetEnd=409 PartOfSpeech=DT] [Text=clothes CharacterOffsetBegin=410 CharacterOffsetEnd=417 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=418 CharacterOffsetEnd=422 PartOfSpeech=WDT] [Text=were CharacterOffsetBegin=423 CharacterOffsetEnd=427 PartOfSpeech=VBD] [Text=worn CharacterOffsetBegin=428 CharacterOffsetEnd=432 PartOfSpeech=VBN] [Text=at CharacterOffsetBegin=433 CharacterOffsetEnd=435 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=436 CharacterOffsetEnd=440 PartOfSpeech=DT] [Text=time CharacterOffsetBegin=441 CharacterOffsetEnd=445 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=446 CharacterOffsetEnd=449 PartOfSpeech=CC] [Text=hearing CharacterOffsetBegin=450 CharacterOffsetEnd=457 PartOfSpeech=VBG] [Text=the CharacterOffsetBegin=458 CharacterOffsetEnd=461 PartOfSpeech=DT] [Text=same CharacterOffsetBegin=462 CharacterOffsetEnd=466 PartOfSpeech=JJ] [Text=old CharacterOffsetBegin=467 CharacterOffsetEnd=470 PartOfSpeech=JJ] [Text=urban CharacterOffsetBegin=471 CharacterOffsetEnd=476 PartOfSpeech=JJ] [Text=legends CharacterOffsetBegin=477 CharacterOffsetEnd=484 PartOfSpeech=NNS] [Text=such CharacterOffsetBegin=485 CharacterOffsetEnd=489 PartOfSpeech=JJ] [Text=as CharacterOffsetBegin=490 CharacterOffsetEnd=492 PartOfSpeech=IN] [Text=was CharacterOffsetBegin=493 CharacterOffsetEnd=496 PartOfSpeech=VBD] [Text=told CharacterOffsetBegin=497 CharacterOffsetEnd=501 PartOfSpeech=VBN] [Text=on CharacterOffsetBegin=502 CharacterOffsetEnd=504 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=505 CharacterOffsetEnd=508 PartOfSpeech=DT] [Text=overnight CharacterOffsetBegin=509 CharacterOffsetEnd=518 PartOfSpeech=JJ] [Text=canoe CharacterOffsetBegin=519 CharacterOffsetEnd=524 PartOfSpeech=NN] [Text=trip CharacterOffsetBegin=525 CharacterOffsetEnd=529 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=529 CharacterOffsetEnd=530 PartOfSpeech=.] 
root(ROOT-0, enjoy-2)
nsubj(enjoy-2, I-1)
xcomp(enjoy-2, seeing-3)
det(clothes-5, the-4)
dobj(seeing-3, clothes-5)
nsubjpass(worn-8, that-6)
auxpass(worn-8, were-7)
rcmod(clothes-5, worn-8)
prep(worn-8, at-9)
det(time-11, this-10)
pobj(at-9, time-11)
cc(worn-8, and-12)
conj(worn-8, hearing-13)
det(legends-18, the-14)
amod(legends-18, same-15)
amod(legends-18, old-16)
amod(legends-18, urban-17)
dobj(hearing-13, legends-18)
mwe(as-20, such-19)
nsubjpass(told-22, as-20)
auxpass(told-22, was-21)
rcmod(legends-18, told-22)
prep(told-22, on-23)
det(trip-27, the-24)
amod(trip-27, overnight-25)
nn(trip-27, canoe-26)
pobj(on-23, trip-27)
punct(enjoy-2, .-28)

Sentence #6 (10 tokens):
The film is also full of funny one liners.
[Text=The CharacterOffsetBegin=535 CharacterOffsetEnd=538 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=539 CharacterOffsetEnd=543 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=544 CharacterOffsetEnd=546 PartOfSpeech=VBZ] [Text=also CharacterOffsetBegin=547 CharacterOffsetEnd=551 PartOfSpeech=RB] [Text=full CharacterOffsetBegin=552 CharacterOffsetEnd=556 PartOfSpeech=JJ] [Text=of CharacterOffsetBegin=557 CharacterOffsetEnd=559 PartOfSpeech=IN] [Text=funny CharacterOffsetBegin=560 CharacterOffsetEnd=565 PartOfSpeech=JJ] [Text=one CharacterOffsetBegin=566 CharacterOffsetEnd=569 PartOfSpeech=CD] [Text=liners CharacterOffsetBegin=570 CharacterOffsetEnd=576 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=576 CharacterOffsetEnd=577 PartOfSpeech=.] 
root(ROOT-0, full-5)
det(film-2, The-1)
nsubj(full-5, film-2)
cop(full-5, is-3)
advmod(full-5, also-4)
prep(full-5, of-6)
amod(liners-9, funny-7)
num(liners-9, one-8)
pobj(of-6, liners-9)
punct(full-5, .-10)

Sentence #7 (21 tokens):
I highly recommend this movie to anyone that was a teenager in the late 70's to early 80's.
[Text=I CharacterOffsetBegin=582 CharacterOffsetEnd=583 PartOfSpeech=PRP] [Text=highly CharacterOffsetBegin=584 CharacterOffsetEnd=590 PartOfSpeech=RB] [Text=recommend CharacterOffsetBegin=591 CharacterOffsetEnd=600 PartOfSpeech=VB] [Text=this CharacterOffsetBegin=601 CharacterOffsetEnd=605 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=606 CharacterOffsetEnd=611 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=612 CharacterOffsetEnd=614 PartOfSpeech=TO] [Text=anyone CharacterOffsetBegin=615 CharacterOffsetEnd=621 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=622 CharacterOffsetEnd=626 PartOfSpeech=WDT] [Text=was CharacterOffsetBegin=627 CharacterOffsetEnd=630 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=631 CharacterOffsetEnd=632 PartOfSpeech=DT] [Text=teenager CharacterOffsetBegin=633 CharacterOffsetEnd=641 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=642 CharacterOffsetEnd=644 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=645 CharacterOffsetEnd=648 PartOfSpeech=DT] [Text=late CharacterOffsetBegin=649 CharacterOffsetEnd=653 PartOfSpeech=JJ] [Text=70 CharacterOffsetBegin=654 CharacterOffsetEnd=656 PartOfSpeech=CD] [Text='s CharacterOffsetBegin=656 CharacterOffsetEnd=658 PartOfSpeech=POS] [Text=to CharacterOffsetBegin=659 CharacterOffsetEnd=661 PartOfSpeech=TO] [Text=early CharacterOffsetBegin=662 CharacterOffsetEnd=667 PartOfSpeech=JJ] [Text=80 CharacterOffsetBegin=668 CharacterOffsetEnd=670 PartOfSpeech=CD] [Text='s CharacterOffsetBegin=670 CharacterOffsetEnd=672 PartOfSpeech=POS] [Text=. CharacterOffsetBegin=672 CharacterOffsetEnd=673 PartOfSpeech=.] 
root(ROOT-0, recommend-3)
nsubj(recommend-3, I-1)
advmod(recommend-3, highly-2)
det(movie-5, this-4)
dobj(recommend-3, movie-5)
prep(recommend-3, to-6)
pobj(to-6, anyone-7)
nsubj(teenager-11, that-8)
cop(teenager-11, was-9)
det(teenager-11, a-10)
rcmod(anyone-7, teenager-11)
prep(teenager-11, in-12)
det(70-15, the-13)
amod(70-15, late-14)
pobj(in-12, 70-15)
possessive(70-15, 's-16)
prep(70-15, to-17)
pobj(to-17, early-18)
dep(early-18, 80-19)
possessive(80-19, 's-20)
punct(recommend-3, .-21)

